examples/subsample.py

```diff
@@ -68,7 +68,7 @@ def query(
 sub = Subsample()
 cmap = mpl.colormaps["coolwarm"]
 norm = Normalize(-2.2, 2.2)
-im = ImageWrapper(sub, {"image": lambda image: cmap(norm(image))})
+im = ImageWrapper(sub)
```

Comment:
This looks broken?

Reply:
fixed, the `ImageWrapper` class takes cmap/norm as inputs, such that you do not need to manually do the func colormapping lambda, just neglected to pass those in in this example.

(It _worked_ by some definition of "work", just with viridis so the cmap/norm setup here was not used, updated now it uses those, just with nicer API for the common need of ImageWrapper)